src/System.IO.FileSystem/System.IO.FileSystem.sln

```diff
@@ -1,26 +1,48 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.21005.1
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
```

Comment:
It doesn't matter- it won't block on 12 and accurately reflects the last VS version that changed the file. The MinimumVisualStudioVersion is still 10 (see below).